TIMER_Sandro.pm

```diff
@@ -549,7 +551,7 @@ sub Timer_FW_Detail($$$$) {
 	$html.= "</tr>";
 
 	for(my $zeile = 0; $zeile < $Timers_Count; $zeile++) {
-		$style_background = "background-color:#000000;" if ($zeile % 2 == 0);
+		$style_background = "background-color:#$Cell_Color;" if ($zeile % 2 == 0);
 		$style_background = "" if ($zeile % 2 != 0);
```

Comment:
Ich denke, wir überlassen die Hintergrundfarbe besser bei den vom Style vorgegeben Werten:
$html.= sprintf("<tr class=\"%s\">", ($zeile & 1)?"odd":"even");

Reply:
OK

Reply:
Jetzt muss ich erstmal hier alles auseinader nehmen vom PR ;-)
Wir wollen ja ungern nicht verschiedene Dinge vermischen.

1) Ich würde das mit der Sonnenauf bzw. Untergang in einem Attribut versehen was im Standardfall aus ist und beim setzen greift mit den Usereingaben um die Fälle 

> REAL,CIVIL,NAUTIC,ASTRONOMIC 

abzudecken?

2) StyleColor Hintergrund, hier würde ich im ersten Schritt erstmal den Code so abändern, das der User die Zeilenunterscheidung im jeweilig gewählten Styleformat sieht und wir nicht die Farbe fest legen.

Als Option, kann man gern noch ein Attribut ergänzen um eine feste Farbe selbst zu definieren bzw. um die Farbe "Zeilenunterscheidung" abzuschalten.

@elektron-bbs siehst du dies ebenso wie ich?
@sepultura30 das müsste doch dann auch so deinem Vorschlag deckeln?